ticket.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+from jiralib import Jira
+
+# how many tickets can be opened on each cycle
+FLOOD_PROTECTION_LIMIT = 3
+
+
+class Ticket:
```

Comment:
I do not understand what this represents.

Reply:
encapsulates the higher level of ticker creation:
ticket create and flood protection,
can be flatten if needed